web/src/main/java/de/helfenkannjeder/helfomat/controller/PictureController.java

```diff
@@ -0,0 +1,49 @@
+package de.helfenkannjeder.helfomat.controller;
+
+import de.helfenkannjeder.helfomat.domain.PictureId;
+import de.helfenkannjeder.helfomat.service.PictureService;
+import org.springframework.core.io.InputStreamResource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+/**
+ * @author Valentin Zickner
+ */
+@RestController
+public class PictureController {
+
+    private final PictureService pictureService;
+
+    public PictureController(PictureService pictureService) {
+        this.pictureService = pictureService;
+    }
+
+    @GetMapping("/picture/{pictureId}/{size}")
+    public ResponseEntity<InputStreamResource> downloadPicture(@PathVariable PictureId pictureId, @PathVariable String size) throws IOException {
```

Comment:
Should we add validation for the size string on controller level? Or map the IllegalArgumentException thrown by the Service to a BadRequest.

Reply:
I added a global exception handler to handle the IllegalArgumentException. It's difficult to validate the size here, since the sizes are based on configuration properties.